src/Hl7.Fhir.ElementModel/IXHtml.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Hl7.Fhir.ElementModel
+{
+    public interface IXHtml
+    {
+        string OriginalElement { get; }
```

Comment:
Also this was just a blabla name. I think we should call it `XHtmlText`

Reply:
Renamed it.